src/index.js

```diff
@@ -62,44 +61,39 @@ class HookActions {
    *   }
    * }
    *
-   * @param {function} actionKey
+   * @param {function} action
+   * @param {string} actionKey
    */
-  createAction(actionKey) {
+  createAction(action, actionKey) {
```

Comment:
Since we are renaming things to make this code easier to reason about, I propose we change `action` here.

Technically, the `contract` a dev is passing to the governor describes functions that return reducers. Under the hood, we transform the contract's functions into `action`s and their invocation results in state being reduced by the function that function returns.

Visually:
```JS
const contract = {
  createReducer() {
    // This returns a reducer.
    return () => ({
      ...this.state
    });
  }
};
```

So, with the above example, the first argument to `createAction` is actually `function createReducer()`. Now, in practice no one will use it this way and we rely on the name of that function as `actionKey` to name our governor's actions, but for our purposes it illustrates my point, I think. Side note: in the case of `async createReducer()` it returns a promise that will create a reducer once fulfilled.

I don't know what to call it - `createReducer` is probably the most accurate, but feels odd... but at least line 69 would look more correct to my eye:

```JS
const reducerOrPromise = createReducer.apply(this, [...args]);
```

That reads very nicely - the invocation of `createReducer` should return a `reducerOrPromise`.